tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "include": ["src/**/*.ts"],
   "compilerOptions": {
     "target": "ES2015",
+    "noEmit": true,
```

Comment:
Will this work for creating the `dist/` folder that is needed when publishing the package?